webapp-django/crashstats/supersearch/views.py

```diff
@@ -46,7 +46,6 @@
 # Facetting on those fields doesn't provide useful information.
 EXCLUDED_FIELDS_FROM_FACETS = (
     'date',
-    'dump',
```

Comment:
Completely unrelated to this work but the dump field doesn't exist anymore. So I just removed it from there. It's highly innocuous.